File: Develop/Backend/src/main/java/com/hientran/sohebox/sco/YoutubeChannelSCO.java
package com.hientran.sohebox.sco;

import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@EqualsAndHashCode(callSuper = true)
public class YoutubeChannelSCO extends BaseSCO {
	private SearchNumberVO id;
	private SearchTextVO channelId;
	private SearchTextVO name;
	private SearchTextVO description;
	private SearchNumberVO categoryId;
	private SearchNumberVO userId;
}

File: Develop/Frontend/src/app/scos/youtubeChannelSCO.ts
import { BaseSCO } from './baseSCO';
import { SearchNumber } from './core_sco/searchNumber';
import { SearchText } from './core_sco/searchText';

export class YoutubeChannelSCO extends BaseSCO {
  id: SearchNumber;
  channelId: SearchText;
  name: SearchText;
  description: SearchText;
  categoryId: SearchNumber;
  userId: SearchNumber;
}
